src/Forms/GridField/GridFieldPaginator.php

```diff
@@ -142,9 +142,10 @@ protected function getGridPagerState(GridField $gridField)
     {
         $state = $gridField->State->GridFieldPaginator;
 
-        // Force the state to the initial page if none is set
```

Comment:
By setting $this->defaults based on the 2nd argument of getState(), this should be unnecessary. I'd suggest that we revert this lines mainly to confirm that the old way of setting a default still works.

Reply:
Scratch this